pkg/chaosdaemon/stress_server_linux.go

```diff
@@ -33,16 +33,67 @@ import (
 func (s *DaemonServer) ExecStressors(ctx context.Context,
 	req *pb.ExecStressRequest) (*pb.ExecStressResponse, error) {
 	log.Info("Executing stressors", "request", req)
-	pid, err := s.crClient.GetPidFromContainerID(ctx, req.Target)
+
+	// CpuStressors
+	cpuInstance, cpuStartTime, err := s.ExecCpuStressors(ctx, req)
 	if err != nil {
 		return nil, err
 	}
-	control, err := cgroups.Load(daemonCgroups.V1, daemonCgroups.PidPath(int(pid)))
+
+	// memoryStressor
+	memoryInstance, memoryStartTime, err := s.ExecMemoryStressors(ctx, req)
 	if err != nil {
 		return nil, err
 	}
 
-	processBuilder := bpm.DefaultProcessBuilder("stress-ng", strings.Fields(req.Stressors)...).
+	return &pb.ExecStressResponse{
+		CpuInstance:     cpuInstance,
+		CpuStartTime:    cpuStartTime,
+		MemoryInstance:  memoryInstance,
+		MemoryStartTime: memoryStartTime,
+	}, nil
+}
+
+func (s *DaemonServer) CancelStressors(ctx context.Context,
+	req *pb.CancelStressRequest) (*empty.Empty, error) {
+	CpuPid, err := strconv.Atoi(req.CpuInstance)
+	if err != nil {
+		return nil, err
+	}
+
+	MemoryPid, err := strconv.Atoi(req.MemoryInstance)
+	if err != nil {
+		return nil, err
+	}
+
+	log.Info("Canceling stressors", "request", req)
+
+	err = s.backgroundProcessManager.KillBackgroundProcess(ctx, CpuPid, req.CpuStartTime)
+	if err != nil {
+		return nil, err
+	}
+
+	err = s.backgroundProcessManager.KillBackgroundProcess(ctx, MemoryPid, req.MemoryStartTime)
+	if err != nil {
+		return nil, err
+	}
+
+	log.Info("killing stressor successfully")
+	return &empty.Empty{}, nil
+}
+
+func (s *DaemonServer) ExecCpuStressors(ctx context.Context,
```

Comment:
s/ExecCpuStressors/ExecCPUStressors/g

Reply:
>When using acronyms, use Pascal case or camel case for acronyms more than two characters long. For example, use HtmlButton or htmlButton. However, you should capitalize acronyms that consist of only two characters, such as System.IO instead of System.Io.(https://stackoverflow.com/questions/15526107/acronyms-in-camelcase#:~:text=Some%20guidelines%20Microsoft%20has%20written%20about%20camelCase%20are%3A)


I looked up the acronym using guideline of camelCase and only the first character should be capitalized in the example of "Cpu" in `ExecCpuStressors`.

Reply:
Aha, go has its own naming convention, https://github.com/golang/go/wiki/CodeReviewComments#initialisms.

And we have talked about this before, https://github.com/chaos-mesh/chaos-mesh/issues/2014

Reply:
Just a mention, a bad example of MS's naming convention is `XMLHttpRequest`. 🌚

Reply:
GOT IT, I have changed `ExecCpuStressors` to `ExecCPUStressors`. 
But I find a lot of generated code still using identifier like CpuXXXXX. 
>Code generated by the protocol buffer compiler is exempt from this rule. 

It is strange that generated code is not of the same standard as the human written one, although it may be diffcult for generator to identify acronyms.🌚